src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -210,7 +210,7 @@ internal void InvalidatePaint()
 		InvalidateParentChildrenPicture();
 	}
 
-	private void InvalidateParentChildrenPicture()
+	internal void InvalidateParentChildrenPicture()
```

Comment:
This could probably be `private protected`.

Reply:
you are right,
but then similar dirty/invalidate methods are all marked `internal` so ill keep it the same here